src/app/functions/universal/dataBrokerNames.ts

```diff
@@ -0,0 +1,199 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export const dataBrokerNamesRecord: Record<string, string> = {
+  "411locate.com": "411 Locate",
+  "peoplefinders.com": "People Finders",
+  "peoplesearchnow.com": "People Search Now",
```

Comment:
You'll have my undying love if we can A-Z these keys to make it easier to diff vs OneRep's list.

Reply:
ONE SEC!

Reply:
Undying love unlocked! ❤️